flink-ml-lib/src/test/java/org/apache/flink/ml/classification/KnnTest.java

```diff
@@ -121,8 +124,9 @@ private static void verifyPredictionResult(Table output, String labelCol, String
                                     @Override
                                     public Tuple2<Double, Double> map(Row row) {
                                         return Tuple2.of(
-                                                (Double) row.getField(labelCol),
-                                                (Double) row.getField(predictionCol));
+                                                ((Number) row.getField(labelCol)).doubleValue(),
```

Comment:
The test code basically should mimic the user code's beahvior.

Since we have updated the stage to output double-typed label column, do we still need to do type conversion here using `Number::doubleValue()`?

Same for prediction column.

Same for `LinearSVCTest`.

Reply:
Since the values of the output table's label column have the same type as the input table's label column specified by the user, the user should already know that this column has double values, and therefore would not need to explicitly convert this column to double type, right?

Would it be simpler to skip type-casting for the label column as well?

Reply:
According to offline discussion, label column is inherited from input table and remain untouched, which means it could be double or integer type. Thus we still need to use (Number)...doubleValue() in order to extract its value.